crates/edr_eth/Cargo.toml

```diff
@@ -18,8 +18,7 @@ reqwest = { version = "0.11", features = ["blocking", "json"] }
 reqwest-middleware = { version = "0.2.4", default-features = false }
 reqwest-retry = { version = "0.3.0", default-features = false }
 reqwest-tracing = { version = "0.4.7", default-features = false, optional = true }
-revm-primitives = { git = "https://github.com/Wodann/revm", rev = "d6fea43", version = "1.3", default-features = false }
-# revm-primitives = { path = "../../../revm/crates/primitives", version = "1.3", default-features = false }
+revm-primitives = { git = "https://github.com/bluealloy/revm", rev = "f36f90e", version = "2.1", default-features = false }
```

Comment:
Is the `2.1` version correct?

Reply:
See https://github.com/NomicFoundation/hardhat/pull/4968#discussion_r1516587639